projects/igniteui-angular/src/lib/grids/tree-grid/tree-grid.component.ts

```diff
@@ -679,6 +766,14 @@ export class IgxTreeGridComponent extends IgxGridBaseDirective implements GridTy
         }
         this.selectionService.clearHeaderCBState();
         this._pipeTrigger++;
+        if (this.rowSelection === 'multipleCascade') {
```

Comment:
Checks should be performed against the enum, rather than against the string representation. This way we can change the string without having to find and replace it everywhere!

Reply:
Agreed, while it's technically safe, it's better if we refer to those values through the exported const if we ever have to rename something.